app/views/includes/govuk_wrapper.scala.html

```diff
@@ -52,14 +57,14 @@
 
 @govukLayout(
     pageTitle = Some(pageTitle),
-    headBlock = Some(hmrcHead(head)),
-    headerBlock = Some(hmrcStandardHeader(
-        serviceUrl = None,
-        serviceName = Some(messages("service.name")),
-        displayHmrcBanner = false
-    )),
+    headBlock = Some(hmrcInternalHead(head, nonce = CSPNonce.get)),
+    headerBlock = Some(hmrcInternalHeader(InternalHeader(
+                    homepageUrl = "/",
+                    serviceName = Some(messages("service.name")),
+                    serviceUrl = "",
```

Comment:
or here this should be relative url to the app or something, now it does nothing but refresh current page

Reply:
see above vid both nav back to dashboard
![Screenshot 2022-12-05 at 12 44 29](https://user-images.githubusercontent.com/47248104/205640293-753d68e1-1fe6-4ee2-b126-a651c8f514c2.png)
![Screenshot 2022-12-05 at 12 44 50](https://user-images.githubusercontent.com/47248104/205640346-f1308aee-2ffc-40ab-8bca-0cb0fc4e61ae.png)